app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :foods, dependent: :destroy
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
```

Comment:
- [ ] [OPTIONAL] It was difficult to sign in after sign up as email confirmation was not working as intended. I had had to temporary disable this option in order to sign in successfully. Kindly improve upon this aspect.

Reply:
Devise send the same email needed to activate the account or in the new password flow, to the server log, there we can click the link to finish the signup process.

![image](https://user-images.githubusercontent.com/53025062/226099149-9f3e5015-88be-4fc5-ace4-5fba022315cb.png)